app/views/practices/search_partials/_search_filters.html.erb

```diff
@@ -1,85 +1,93 @@
-    <legend class="usa-sr-only">Tag Filters</legend>
-    <div class="bg-gray-5 desktop:padding-bottom-2px search-category-filters">
-      <h2 class="usa-accordion__heading">
-        <button class="usa-accordion__button mobile-category-accordion desktop:display-none"
-                aria-expanded="false"
-                type="button">
-          Tags
-        </button>
-      </h2>
+<h3 class="padding-bottom-1"> Refine Results </h3>
 
-      <div class="grid-row category-container display-none padding-top-2 desktop:padding-bottom-1 desktop:padding-top-0 desktop:display-flex desktop:padding-x-2">
-        <p class="text-bold line-height-26 margin-y-2 grid-col-12 display-none desktop:display-block margin-top-2">
-        <a href="#parent-category-modal" class="dotted parent-category-modal cursor-pointer margin-bottom-2" aria-controls="parent-category-modal" data-open-modal>
-          <span class="usa-sr-only">Open parent tag definitions modal</span>
-          Tags
-        </a>
-          <%= render partial: 'shared/parent_category_modal' %>
-        </p>
-        <% @parent_categories.each do |parent_category, sub_categories| %>
-          <%
-            if sub_categories.any?
-              categories = [{name: "All #{parent_category.name.downcase}", parent_category: parent_category, id: parent_category.name}] + sub_categories
-            else
-              categories = []
-            end
-            sliced_categories = split_data_into_three_columns(categories)
-          %>
-          <% if sub_categories.any? %>
-            <div class="grid-row grid-col-12 category-container display-none padding-top-2 desktop:padding-bottom-1 desktop:padding-top-0 desktop:display-flex padding-left-0 padding-right-0">
-              <p class="line-height-26 margin-y-2 grid-col-12 display-none desktop:display-block desktop:margin-top-1 desktop:margin-bottom-0"><%= parent_category.name %></p>
-              <% sliced_categories.each_with_index do |sc, i| %>
-                <%= render partial: 'practices/search_partials/category_filter_column', locals: { categories: sc, column: i, parent_category: parent_category } unless i === 3 %>
-              <% end %>
+<div id="search-sort-options-container" class="padding-bottom-2" aria-label="Sort options">
+  <%
+    sort_options = [
+      ['Sort by most relevant', 'most_relevant'],
+      ['Sort by A to Z', 'a_to_z'],
+      ['Sort by most adopted innovations', 'adoptions'],
+      ['Sort by most recently added', 'added']
+    ]
+  %>
+  <h4 class="text-bold">Sort By</h3>
+  <% sort_options.each do |option| %>
+    <div class="usa-radio">
+      <input id="search_sort_option_<%= option[1] %>" class="usa-radio__input" type="radio" name="search_sort_option" value="<%= option[1] %>" <%= 'checked' if option[1] == 'adoptions' %> aria-label="<%= option[0] %>">
+      <label class="usa-radio__label" for="search_sort_option_<%= option[1] %>" aria-label="<%= option[0] %>">
+        <%= option[0] %>
+      </label>
```

Comment:
```suggestion
      <input id="search_sort_option_<%= option[1] %>" class="usa-radio__input" type="radio" name="search_sort_option" value="<%= option[1] %>" <%= 'checked' if option[1] == 'adoptions' %>>
      <label class="usa-radio__label" for="search_sort_option_<%= option[1] %>">
        <%= option[0] %>
      </label>
```
The aria-labels here can be removed since they're duplicating content. Also consider using `fieldset` and `legend` here to get the radio buttons semantically grouped nicely: https://designsystem.digital.gov/components/radio-buttons/

Reply:
@camillevilla I actually consciously chose not to make it a fieldset because from what I read a fieldset is intended to be part of a form, which the sort options are not, is it ok to make them a fieldset you think?

Reply:
@PhilipDeFraties ahh ok I see what you're saying re: not-form-ness here. The `Sort by` options immediately update the results. In that case I'm thinking we might need to add some `aria` to indicate that results immediately changed...and... yeah let's do that in another PR discussion.